docs/local-overlay.yaml

```diff
@@ -1,18 +1,13 @@
 description: |
   A local overlay to deploy the charms from this repo. This is meant for developers.
-  The appropriate steps to use this would be to :
+  The appropriate steps to use this would be to:
   1) pull the repo locally
   2) cd metallb-operator
-  2) build the charms with `tox -e build` 
+  2) build the charms with `make charms` 
   3) deploy with the overlay: 
-      `juju deploy ./bundle.yaml --overlay ./docs/local-overlay.yaml`
-bundle: kubernetes
+      `juju deploy ./bundle --overlay ./docs/local-overlay.yaml`
 applications:
   metallb-controller:
-    charm: ../charms/metallb-controller/build/metallb-controller.charm
-    resources:
-      metallb-controller-image: 'metallb/controller:v0.9.3'
+    charm: ./build/metallb-controller.charm
```

Comment:
Ah, I think I was being overly optimistic and not wanting to have it hard-coded in multiple places.  It may be that once the charms are in the store with resources attached, we might be able to away with overriding the charm without specifying the resources, but I kind of think not, since generally when deploying a local charm you have to provide all of the resources. (There's a small chance that since Juju knows from the original bundle where the charm would have come from in the store, it might be smart enough to pick the resources up from there, but I wouldn't bet on it.)

Reply:
also link location will need to be ../build/metallb-controller.charm with the new 'make build'. Tests are failing because this points to /docs/build/metallb-speaker.charm (since the overlay is in docs)